charts/csi-baremetal-driver/values.yaml

```diff
@@ -124,4 +125,3 @@ fluentbitAgent:
 
 alerts:
   deployConfig: false
-  
```

Comment:
Leave an empty string at the end of the file

Reply:
In original file this line contains space, in my - not.

Reply:
It's better to have an empty line at the of the file https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795

Reply:
Discussed in slack